src/Akeneo/Tool/Component/Batch/Job/Job.php

```diff
@@ -270,6 +272,11 @@ protected function handleStep(StepInterface $step, JobExecution $jobExecution)
                 $step->setStoppable($this->isStoppable);
             }
 
+            if ($step instanceof TrackableStepInterface || $step instanceof TrackableTaskletInterface) {
```

Comment:
The Tasklet and step are not the same thing. 
There is a TaskletStep that is a step and launch a Tasklet

`class TaskletStep extends AbstractStep implements TrackableStepInterface`

```suggestion
            if ($step instanceof TrackableStepInterface) {
```